lib/get-api.js

```diff
@@ -3,8 +3,8 @@ const { trim } = require('omnibelt');
 
 process.env.LOSANT_API_URL = process.env.LOSANT_API_URL || 'https://api.losant.com';
```

Comment:
whats the difference between `LOSANT_API_URL` and `DEFAULT_API_URL`?

Reply:
iirc the idea was to migrate creds using `api.losant.com` in all cases, and `DEFAULT_API_URL` exists for testing with that. 
That being said, we could migrate creds using whatever the user has `LOSANT_API_URL` set to (and remove `DEFAULT_API_URL`) which seems like a better solution to me.